Git error on startup

Just pull the git and build the app.
When I launch it, got an error about repository not found : https://github.com/PlayOnLinux/Oldwares
```
[ERROR] org.phoenicis.repository.repositoryTypes.Repository (l.51) - Fetching installable applications failed!
org.phoenicis.repository.RepositoryException: Folder '/home/tutul/.Phoenicis/cache/git592808621' is no git-repository
	at org.phoenicis.repository.repositoryTypes.GitRepository.fetchInstallableApplications(GitRepository.java:114)
	at java.util.stream.Collectors.lambda$toConcurrentMap$59(Collectors.java:1473)
	at java.util.stream.ReferencePipeline.lambda$collect$1(ReferencePipeline.java:496)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
	at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291)
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinTask.doInvoke(ForkJoinTask.java:401)
	at java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:734)
	at java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:160)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:174)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:583)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:496)
	at org.phoenicis.repository.repositoryTypes.MultipleRepository.fetchInstallableApplications(MultipleRepository.java:54)
	at org.phoenicis.repository.repositoryTypes.CachedRepository.fetchInstallableApplications(CachedRepository.java:36)
	at org.phoenicis.repository.repositoryTypes.Repository.fetchInstallableApplications(Repository.java:49)
	at org.phoenicis.repository.repositoryTypes.BackgroundRepository.lambda$fetchInstallableApplications$0(BackgroundRepository.java:51)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.eclipse.jgit.api.errors.TransportException: https://github.com/PlayOnLinux/Oldwares: Authentication is required but no CredentialsProvider has been registered
	at org.eclipse.jgit.api.FetchCommand.call(FetchCommand.java:224)
	at org.eclipse.jgit.api.CloneCommand.fetch(CloneCommand.java:216)
	at org.eclipse.jgit.api.CloneCommand.call(CloneCommand.java:137)
	at org.phoenicis.repository.repositoryTypes.GitRepository.fetchInstallableApplications(GitRepository.java:94)
	... 25 common frames omitted
Caused by: org.eclipse.jgit.errors.TransportException: https://github.com/PlayOnLinux/Oldwares: Authentication is required but no CredentialsProvider has been registered
	at org.eclipse.jgit.transport.TransportHttp.connect(TransportHttp.java:528)
	at org.eclipse.jgit.transport.TransportHttp.openFetch(TransportHttp.java:326)
	at org.eclipse.jgit.transport.FetchProcess.executeImp(FetchProcess.java:136)
	at org.eclipse.jgit.transport.FetchProcess.execute(FetchProcess.java:122)
	at org.eclipse.jgit.transport.Transport.fetch(Transport.java:1236)
	at org.eclipse.jgit.api.FetchCommand.call(FetchCommand.java:213)
	... 28 common frames omitted
```

--- a/phoenicis-repository/src/main/java/org/phoenicis/repository/FilesystemJsonRepositoryLocationLoader.java
+++ b/phoenicis-repository/src/main/java/org/phoenicis/repository/FilesystemJsonRepositoryLocationLoader.java
@@ -44,10 +44,7 @@ public class FilesystemJsonRepositoryLocationLoader implements RepositoryLocatio
         } else {
             try {
                 result.add(new GitRepositoryLocation.Builder()
-                        .withGitRepositoryUri(new URL("https://github.com/PlayOnLinux/Scripts").toURI())
-                        .withBranch("master").build());
-                result.add(new GitRepositoryLocation.Builder()
-                        .withGitRepositoryUri(new URL("https://github.com/PlayOnLinux/Oldwares").toURI())
+                        .withGitRepositoryUri(new URL("https://github.com/PlayOnLinux/Scripts").toURI()) //FIXME Change to use PhoenicisOrg instead if PlayOnLinux ?
                         .withBranch("master").build());
                 result.add(new ClasspathRepositoryLocation("/org/phoenicis/repository"));
             } catch (URISyntaxException | MalformedURLException e) {

--- a/phoenicis-repository/src/main/java/org/phoenicis/repository/FilesystemJsonRepositoryLocationLoader.java
+++ b/phoenicis-repository/src/main/java/org/phoenicis/repository/FilesystemJsonRepositoryLocationLoader.java
@@ -44,7 +44,7 @@ public class FilesystemJsonRepositoryLocationLoader implements RepositoryLocatio
         } else {
             try {
                 result.add(new GitRepositoryLocation.Builder()
-                        .withGitRepositoryUri(new URL("https://github.com/PlayOnLinux/Scripts").toURI()) //FIXME Change to use PhoenicisOrg instead if PlayOnLinux ?
+                        .withGitRepositoryUri(new URL("https://github.com/PhoenicisOrg/Scripts").toURI())
                         .withBranch("master").build());
                 result.add(new ClasspathRepositoryLocation("/org/phoenicis/repository"));
             } catch (URISyntaxException | MalformedURLException e) {

